lib/MiLight/MiLightRadioPL1197_LT8900.cpp

```diff
@@ -0,0 +1,631 @@
+/*
+ * MiLightRadioPL1167_LT89000.cpp
+ *
+ *  Created on: 31 March 2017
+ *      Author: WoodsterDK
+ *
+ *  Very inspired by:
+ *  https://github.com/pmoscetta/authometion-milight/tree/master/Authometion-MiLight
+ *  https://bitbucket.org/robvanderveer/lt8900lib
+ */
+
+#include "MiLightRadioPL1167_LT8900.h"
```

Comment:
The formatting here is a little inconsistent. Can we run some autoformatter with:

* K&R braces (I know this is a personal preference, but would be nice for it to be consistent within the project)
* Two space indents

Obviously I'm happy to do this within a branch if you'd prefer not to.